exercises/concept/short-fibonacci/tests/short-fibonacci.rs

```diff
@@ -18,6 +18,8 @@ fn test_buffer() {
 fn test_fibonacci() {
     let fibb = fibonacci();
     assert_eq!(fibb.len(), 5);
+    assert_eq!(fibb[0], 1);
```

Comment:
```suggestion
```

Omitting this line allows us to support the sequences starting with `[0, 1]` or `[1, 1]`.

Reply:
but it would also permit weird sequence like `[5, 1, 6, 7, 13]` to pass,